app/views/dashboards/analytics.erb

```diff
@@ -1,24 +1,25 @@
-<% title "Pro Analytics" %>
+<% title "Analytics" %>
```

Comment:
I chose to keep this dashboard template/view the exact same and just renamed it from "Pro Analytics" to "Analytics". It seemed a little weird to extract it out into the basic `/dashboard` view, so keeping it as a separate route `/dashboard/analytics` seemed like a smoother transition, IMO.